### PR TITLE
Various changes

### PR DESCRIPTION
**Pull request by funkyfuture**
these are some changes i applied for our deployments. feel free to cherry-pick what you deem useful and/or request improvements.

i don't know what is causing the error in the tests / how to fix it. this isn't happening when using it in our environment (on a fresh machine); prosody is running at that point and the reload is working.

### CODE REVIEW

File: tasks/nginx.yml [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
   template:
     src: nginx/nginx.conf.j2
     dest: /etc/nginx/nginx.conf
     owner: root
     group: root
     mode: "0644"
-  notify: restart nginx
+  notify: reload nginx
+
+- name: Add mimetype for wasm files.
```
Comment (0x46616c6b) [inline tasks/nginx.yml:17]: Why is that needed?

File: templates/logrotate.d/jitsi-videobridge.j2 [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/var/log/jitsi/jvb.log {
+  daily
```
Comment (0x46616c6b) [inline templates/logrotate.d/jitsi-videobridge.j2:2]: The videobridge is rotated hourly in the default and it seems that is needed. The service logs a lot.

File: tasks/nginx.yml (hunk 1 above)
Comment (funkyfuture) [reply tasks/nginx.yml:17]: i observed that the application logged an error in the browser's console that hints on a missing mimetype declaration. i have the *suspicion* that this change made the "Blur my background" functional in my case. but haven't investigated what change exactly fixed that.

setting proper mimetype headers is a good idea in any case.

File: templates/logrotate.d/jitsi-videobridge.j2 (hunk 2 above)
Comment (funkyfuture) [reply templates/logrotate.d/jitsi-videobridge.j2:2]: what about `{ jitsi_meet_logrotate_retained_days *24 }}` then? and related, it seems that `jitsi_meet_videobridge_loglevel` has no effect, doesn't it? i see entries about health checks at info level every ten seconds.

Comment (funkyfuture) [reply templates/logrotate.d/jitsi-videobridge.j2:2]: `/etc/jitsi/videobridge/logging.properties` defines a console logger.

File: tasks/nginx.yml (hunk 1 above)
Comment (t2d) [reply tasks/nginx.yml:17]: I agree, that proper mimetypes can't hurt

**Comment by t2d** [comment]
Secret derivation: I like the idea, but it breaks compatibility with our current way of storing secrets. @0x46616c6b waht do you think about it. 

Prosody uncoditional password change: Looks useful to me, but we can't have a task that changes all the time. 

Prosody reload handler: Would be nice, but did break testing

Logrotate: I think we probably want to bring all this to syslog anyway.

The other things I either completely liked and cherry-picked or disliked.

**Comment by funkyfuture** [comment]
> Prosody uncoditional password change: Looks useful to me, but we can't have a task that changes all the time.

that shouldn't be the case / the derived secrets should be deterministic.

> Logrotate: I think we probably want to bring all this to syslog anyway.

`journald` as a target option would be nice too.

File: templates/logrotate.d/jitsi-videobridge.j2 (hunk 2 above)
Comment (funkyfuture) [reply templates/logrotate.d/jitsi-videobridge.j2:2]: > i see entries about health checks at info level every ten seconds.

i'm being told that this behaviour was a bug and has been fixed in a recent release. but i still see this with the most recent `2.1-197-g38256192-1`. could something from the role be the cause?

**Comment by funkyfuture** [comment]
i'd have some further small tweaks, but i'd rather close this pr as i doubt it'd get any more clearly here. so i'd prepare another pr. but i'm curious why you didn't pick d9d622917d9e33801f4586aef6e909e0e5f4212f, as this was the only change that i'd consider a bug fix.

**Comment by t2d** [comment]
> i'd have some further small tweaks, but i'd rather close this pr as i doubt it'd get any more clearly here. so i'd prepare another pr. but i'm curious why you didn't pick d9d6229, as this was the only change that i'd consider a bug fix.

i just missed it. Fixed now. Yes, please multiple small PRs. Much easier to handle.

File: templates/logrotate.d/jitsi-videobridge.j2 (hunk 2 above)
Comment (t2d) [reply templates/logrotate.d/jitsi-videobridge.j2:2]: > i'm being told that this behaviour was a bug and has been fixed in a recent release. but i still see this with the most recent 2.1-197-g38256192-1. could something from the role be the cause?

does #29 help?

**Comment by funkyfuture** [comment]
i opened two follow-up prs.

File: templates/logrotate.d/jitsi-videobridge.j2 (hunk 2 above)
Comment (funkyfuture) [reply templates/logrotate.d/jitsi-videobridge.j2:2]: that affects jicofo, not the videobridge, doesn't it?

Comment (t2d) [reply templates/logrotate.d/jitsi-videobridge.j2:2]: okay. there are now other diffs to latest upstream release.

Comment (t2d) [reply templates/logrotate.d/jitsi-videobridge.j2:2]: I also incorporated this patch, but without jitsi-videobridge. Their logs are *just to big*
